### PR TITLE
Gra 1011/remove mq dyn sec

### PR DESCRIPTION
**Pull request by abhishek9686**
Testing Steps:
Related Client Side PR: https://github.com/gravitl/netclient/pull/186
- [x] use the nm-interactive.sh to install the netmaker
- [x] verify the  mq password and username are asked as input from the user when you run the script
- [x] check whether netmaker server is able to connect to mq
- [x] and no errors on mq container logs

### CODE REVIEW

File: compose/docker-compose.yml [hunk 1]
```diff
@@ -28,7 +27,8 @@ services:
       MQ_PORT: "443"
       MQ_SERVER_PORT: "1883"
       VERBOSITY: "1"
-      MQ_ADMIN_PASSWORD: "REPLACE_MQ_ADMIN_PASSWORD"
+      MQ_PASSWORD: "REPLACE_MQ_PASSWORD"
```
Comment (mattkasun) [inline compose/docker-compose.yml:30]: these changes are also applicable to docker-compose.ee.yml and docker-compose.reference.yml

**Review by mattkasun — CHANGES_REQUESTED** [review]

File: compose/docker-compose.yml (hunk 1 above)
Comment (theguy951357) [reply compose/docker-compose.yml:30]: that explains why my setup isn't working. I am testing on ee.

Comment (theguy951357) [reply compose/docker-compose.yml:30]: docker-compose.ee.yml actually needs a few things updated. there's no stun setup on there either.

**Review by theguy951357 — APPROVED** [review]
approved pending changes addressed.

File: compose/docker-compose.yml (hunk 1 above)
Comment (abhishek9686) [reply compose/docker-compose.yml:30]: updated docker-compose.ee.yml and I believe docker-compose.reference.yml is being updated by @afeiszli

**Review by 0xdcarns — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Pull request merged**